### ISSUE TITLE
get crash when open map

### ISSUE THREAD

**Issue by tyzero** [phunware/maas-mapping-android-sdk#44]
I'm using the latest version
```java
  compile("com.phunware.mapping:mapping:3.3.0")
  compile ("com.phunware.location:provider-managed:3.2.2")
```

there is the building_id my app using :
53866

when open phunware map, it would get this crash sometime, randomly

```java
java.lang.NullPointerException: Attempt to invoke virtual method 'boolean java.lang.String.equals(java.lang.Object)' on a null object reference
        at com.phunware.mapping.manager.NetworkSegment.equals(NetworkSegment.java:57)
        at java.util.ArrayList.indexOf(ArrayList.java:331)
        at java.util.ArrayList.contains(ArrayList.java:314)
        at com.phunware.mapping.manager.NavigationHelper.setBuildingData(NavigationHelper.java:324)
        at com.phunware.mapping.manager.PhunwareMapManager.onBuilding(PhunwareMapManager.java:449)
        at com.phunware.mapping.manager.PhunwareMapManager.access$200(PhunwareMapManager.java:74)
        at com.phunware.mapping.manager.PhunwareMapManager$1.onResponse(PhunwareMapManager.java:351)
        at okhttp3.RealCall$AsyncCall.execute(RealCall.java:153)
        at okhttp3.internal.NamedRunnable.run(NamedRunnable.java:32)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1162)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:636)
        at java.lang.Thread.run(Thread.java:764)
```

**Comment by astolzberg** [comment]
@tyzero Can you try it with the latest Location version (3.3.1)? 
https://github.com/phunware/maas-location-android-sdk

**Comment by tyzero** [comment]
@astolzberg ok, thanks.

**Comment by astolzberg** [comment]
We recently released Location and Mapping SDKs v3.4.0 -- I'd recommend trying with those if you are still having issues.

**Comment by avakkachen** [comment]
Closing this issue. Will re-open if reproducible in latest version of Mapping SDK 3.9.9
